components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -751,12 +770,237 @@ impl Interpolate for LengthOrNone {
             }
         } else {
             // TODO(gw): Implement matrix decomposition and interpolation
+            // TODO(canaltinova): Do I need to interpolate each item of the list?
             result.extend_from_slice(from_list);
         }
 
         TransformList(Some(result))
     }
 
+    fn rotate_to_matrix(x: f32, y: f32, z: f32, a: SpecifiedAngle) -> ComputedMatrix {
```

Comment:
Or just leave a spec link to https://drafts.csswg.org/css-transforms/#Rotate3dDefined